CatalogAPI/Data/Context.cs

```diff
@@ -0,0 +1,20 @@
+﻿using CatalogAPI.Data.Model;
+namespace CatalogAPI.Data
+{
+    public static class Context
+    {
+        public static readonly List<Category> Repository = new();
```

Comment:
The "Testability" requirement is not passed. Better extract an interface

Reply:
I created Interfcace